tox.ini

```diff
@@ -2,9 +2,10 @@
 envlist = py27, py32, py33, py34, pypy, flake8, checkspelling
 
 [testenv]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps = -rtest-requirements.txt
 commands = coverage run --source=markdown {toxinidir}/run-tests.py {posargs}
-           coverage report --show-missing
+           coveralls
```

Comment:
Not sure I'm a fan of this. I like looking at the coverage report in the tox logs. In fact, I generally run tox locally and will review the coverage report. With this change I can't do that anymore. And as coverage was run in a now deactivated tox venv, I can't just run `coverage report` manually either. Now I have to manually run `coverge run ...` and `coverage report`  to get coverage reports locally before pushing to GitHub. This is a non-starter for me.

Reply:
Although, is there any reason why we can't leave the `coverage report ` line? Just call `coveralls` after?

Reply:
I personally don't think coveralls should be run in the test suite, but should instead be run by Travis. If I want to run the test suite locally, it's going to try to run coveralls which doesn't make sense to me.

Reply:
@facelessuser good call. Let's not run coveralls from tox. It we can't make it work directly from travis, then we need to switch to another provider. I'll try to find some time later this evening to enable codecov.